admin_guide/transitioning_from_postgresql_to_citus.rst

```diff
@@ -16,4 +16,63 @@ directly pipe it to a copy into the distributed table. For example:
 
 One thing to note as you transition from a single node to multiple nodes is that you should create your extensions, operators, user defined functions, and custom data types on all nodes.
 
-If you have any questions or require assistance in scaling out your existing PostgreSQL installation, please get in touch with us at engage@citusdata.com.
+Multi-Tenant Applications
+-------------------------
+
+Citus is well suited to hosting B2B multi-tenant application data. All tenants share a schema and each tenant identifies its own rows in the tables by a pre-existing or artificially added distribution column, isolating each tenant's data in its own shard. This section will explore how to model for the multi-tenant scenario, including necessary adjustments to the schema and queries.
```

Comment:
"isolating each tenant's data in its own shard." This isn't strictly true. Formally, *all* the data for a tenant will be localized to a single shard, however its not guaranteed that the shard will only contain that tenant. One shard will contain all data for potentially multiple tenants.